models/models.go

```diff
@@ -24,3 +28,224 @@ func standardizeError(err error) error {
 
 	return err
 }
+
+//This is an example of a structured format that is compatible with Object-Relational Mapping (ORM) systems.
+type MyStruct struct {
+	Id          int64  `json:"id" db:"id" primarykey:"true" `
+	Name        string `json:"name" db:"name" `
+	Age         int    `json:"age" db:"age"`
+	Email       string `json:"email" db:"email"`
+	PhoneNumber int64  `json:"phone_number" db:"phone_number"`
+}
+
+//structure variable is having the &struct.please pass the pointer of the structure not the variable
+func FindFirst(tableName string, structure interface{}) error {
+	primaryKeyField, err := PrimaryKeyIdentifier(structure)
```

Comment:
if i need to filter with another column like email, place can you manage it ?

because of this function i think only handling with Primary key column

Reply:
you are thinking in wrong direction for this .
the requirement is only for finding the the first entry of that table by its primary key 
because primary key will be able to determine which entry is first and which is last.